raiutils/requirements.txt

```diff
@@ -1 +1,5 @@
-requests==2.25.1
\ No newline at end of file
+numpy>=1.17.2
```

Comment:
Hmm, I really wanted to keep this package with as few dependencies as possible, so it can be easily integrated into other projects while minimizing conflicts.  I guess these dependencies are unavoidable though.  Probably making them soft dependencies instead of hard dependencies doesn't make sense?

Reply:
I guess I can unpin them so that these don't conflict with our other packages which have dependencies on them. What do you think?

Reply:
Yeah, maybe we can try just unpinning them, so they have minimal conflicts.  Good idea!

Reply:
Done. removed the pins